src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_blocked_opt.cpp

```diff
@@ -422,6 +422,7 @@ static inline int SelectVecSizeFromFormat(const DataTensor& tensor) {
 static inline int GetInnerBatchBlockSize(const DataTensor& tensor) {
     auto layout = tensor.GetLayout();
     switch (layout) {
+    case DataLayout::bfyx:
```

Comment:
I don't think the kernel supports all the shapes/cases with bfyx format? Probably it should be limited only to the cases with `spatial_broadcasting` (in terms of eltwise_kernel_blocked_opt kernel)
Also, please update GetSupportedKey() function accordingly to allow support of planar format not only for the 2nd input